src/Mvc/Mvc.TagHelpers/src/Resources.resx

```diff
@@ -168,4 +168,7 @@
   <data name="PersistComponentStateTagHelper_FailedToInferComponentPersistenceMode" xml:space="preserve">
     <value>The persistence mode could not be determined based on the rendered components. Specify a value using the '{0}' attribute.</value>
   </data>
-</root>
+  <data name="ComponentOutputTagHelper_NoMatchingName" xml:space="preserve">
+    <value>No component has an output name matching '{0}'.</value>
```

Comment:
Is this used anywhere?

Reply:
Good catch, this was used when `<component-output>` was a thing. I'll remove this.